src/index.html

```diff
@@ -2,16 +2,137 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="POST" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend class="legend">
+          Personal information
+        </legend>
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="label">
+          I accept the term of the agreement
+          <input type="checkbox" name="agreementAccept">
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend class="legend">
+          Registration
+        </legend>
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="25"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend class="legend">
+          An interesting fact about you!
+        </legend>
+        <label class="label">
+          Do you have cats?
+          <input
+            type="radio"
+            name="cats"
+            value="Yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="cats"
+            value="No"
+          >
+          No
+        </label>
```

Comment:
![image](https://user-images.githubusercontent.com/80700247/229341940-08f6af24-55be-4b4b-8176-5f8f9dfd2164.png)

No need to set label as wrapper for radio buttons, we'd like to activate each radio by click on 'Yes' or 'No'